husky:
  topic_name: /scan
  queue_size: 1
